client/src/components/dashboard/Dashboard.js

```diff
@@ -1,15 +1,179 @@
+                            <Form.Control type="text" placeholder={"Enter your GitHub Username Here"} />
+                        </Form.Group>
+
+                        <Form.Group controlId="formGithubToken">
+                            <Form.Label>Github Personal Access Token</Form.Label>
+                            <Form.Control type="text" placeholder={"Enter your GitHub Personal Access Token Here"} />
+                        </Form.Group>
+
+                    </Form>
+
+                </Modal.Body>
+
+                <Modal.Footer>
+                    <Button onClick={() => {
+                        handleCloseOne();
+                        handleShowTwo()
+                    }}>
+                        Next
+                            </Button>
+                </Modal.Footer>
+            </Modal>
+            
+            {/*'Selecting Template 1 Modal'*/}
+            <Modal show={show2} onHide={handleCloseTwo}  centered size="lg">
+                <Modal.Header closeButton>
+                    <Modal.Title id="contained-modal-title-vcenter">
+                    Select a template
+                    </Modal.Title>
+                </Modal.Header>
+                <Modal.Body>
+                    <Image src={homeTemplateImg1} className="img-fluid"/>
+                    <p>
+                    Option 1: Light minimalist design with blue and white accents.
+                    </p>
+                </Modal.Body>
+                
+                <Modal.Footer>
+                    <Button variant="secondary"
+                        onClick={() => {
+                        handleCloseTwo();
+                        handleShowThree()}}
+                    >
+                        <BsChevronRight />
+                    </Button>
+                    <Button onClick={() => {dispatch(createWebsite());
+                    handleCloseTwo()}}>
+                        <BsCheck />
+                    </Button>
+                </Modal.Footer>
+            </Modal>
+
+            {/*'Selecting Template 2 Modal'*/}
+            <Modal show={show3} onHide={handleCloseThree}  centered size="lg">
+                <Modal.Header closeButton>
+                    <Modal.Title id="contained-modal-title-vcenter">
+                    Select a template
+                    </Modal.Title>
+                </Modal.Header>
+                <Modal.Body>
+                    <Image src={homeTemplateImg2} className="img-fluid"/>
+                    <p>
+                    Option 2: Dark minimalist design with blue and black accents.
+                    </p>
+                </Modal.Body>
+                
+                <Modal.Footer>
+                    <Button variant="secondary"
+                        onClick={() => {
+                        handleShowTwo();
+                        handleCloseThree()}}
+                    >
+                        <BsChevronLeft />

```

Comment:
These arrow icons arent consistent and informative better to have words like usual.